<launch>
  <node name="usb_cam" pkg="usb_cam" type="camera_image" output="screen" respawn="true" >
    <param name="video_device" value="/dev/video2" />
    <param name="camera_info_url" value="file:///home/moi/.usb_cam.yaml"/>
    <param name="camera_frame_id" value="base_camera_link_usb_cam" />
    <param name="publish_rate" value="10" />
    <param name="image_show" value="false" />
    <param name="image_topic" value="raw_image" />
    <param name="release" value="false" />

  </node>
  <node name="image_view" pkg="image_view" type="image_view" output="screen" respawn="true" >
    <remap from="image" to="/usb_cam/raw_image"/>
  </node>

</launch>
